charm/ksql/reactive/ksql.py

```diff
@@ -97,173 +77,10 @@ def configure_ksql(kafka):
     hookenv.status_set('maintenance', 'setting up ksql')
 
     ksql = Ksql()
-    ksql.configure(kafka.kafkas())
+    ksql.install(kafka_units=kafka.kafkas())
```

Comment:
This function partially duplicates functionality in the configure() function above, but not consistently. I fear that will create subtle hard-to-debug problems in production.

I think it'd be better to react to these flags by causing configure() to fire -- clearing the ksql.configured flag.

Reply:
Also note that this function will probably fire every 5 minutes once all the relations are established. That's OK, but we should only reconfigure & restart ksql if something actually changes.